bin/block-info-tp

```diff
@@ -15,22 +15,26 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
-import os
-import sys
-import sysconfig
-
-build_str = "lib.{}-{}.{}".format(
-    sysconfig.get_platform(),
-    sys.version_info.major, sys.version_info.minor)
-
-sys.path.insert(0, os.path.join(
-    os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
-    'sdk', 'python'))
-sys.path.insert(0, os.path.join(
-    os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
-    'families', 'block_info', 'python'))
-
-from sawtooth_block_info.processor.main import main
+# Copyright 2018 Bitwise IO, Inc.
```

Comment:
This line should be moved under `Copyright 2017 Intel Corporation` and the rest of the license should be removed